### PR TITLE
🌱 Increment internal versions to use release v0.4.3

### PR DESCRIPTION
**Pull request by mjlshen**
This will allows users of this GitHub action to use the changes from #56

### CODE REVIEW

**Review by killianmuldoon — COMMENTED** [review]
/lgtm

Thanks!

**Comment by mjlshen** [comment]
cc @vincepri for approval

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

File: action.yml
Comment (sbueringer) [inline action.yml:12]: I think this should be v0.4.3 now.

We wouldn't want a kube-builder v0.4.3 relesae that points to a v0.4.2 image?

**Comment by killianmuldoon** [comment]
>I think this should be v0.4.3 now.

Only if there's certain to be a release with that version - i.e. this would be a release PR. If we want to use the sha of a commit using the current version with an image that exists is the right approach IMO

File: action.yml
Comment (mjlshen) [reply action.yml:12]: I'm happy to make the change to v0.4.3 and have done so since it's easy enough to revert - adding @killianmuldoon 's point below this PR should probably only be merged if a 0.4.3 release if cut right after, otherwise the v0.4.3 image won't actually exist.

**Comment by sbueringer** [comment]
Ah I see

**Comment by sbueringer** [comment]
/lgtm

@vincepri @camilamacedo86 Can we merge this PR and cut a new v0.4.3 release? So we have a release without the panic and a release which uses the same image (v0.4.3) as the release tag (v0.4.3)

File: verify/go.mod
Comment (furkatgofurov7) [inline verify/go.mod:10]: while we can't bump this to non-existent version as we are doing in other file with image version, would not it be problem (not having our fix included in the tool) when using it from CAPI?

Comment (sbueringer) [reply verify/go.mod:10]: This version is not used at all, see the replace in l.5.

Maybe we should just set it to v0.0.0 (if possible) to avoid confusion

Comment (furkatgofurov7) [reply verify/go.mod:10]: @sbueringer I see, have not expanded the file fully to see the replace.. Then it should be all good 👍🏼

Comment (sbueringer) [reply verify/go.mod:10]: Yup. It's just pretty confusing since we changed this version to something looking almost like it makes sense a few PRs ago... :D

**Review by furkatgofurov7 — COMMENTED** [review]
cc @vincepri PTAL

We (CAPI release team) would appreciate a new release once this is merged, thanks!

**Review by vincepri — APPROVED** [review]
/approve

**Comment by mjlshen** [comment]
@vincepri Thanks for getting this merged in! Could you help get a v0.4.3 release cut as well? Unfortunately the code that got merged won't work without it.

**Pull request merged**
